app/controllers/hirings_controller.rb

```diff
@@ -14,7 +14,8 @@ def create
     vehicle = @hiring.vehicle
     if @hiring.save
       vehicle.decrement! :quantity
-      BillingMailer.send_mail_billing(@hiring, current_user).deliver_now
+      @hiring.update_attributes give_back_time: @hiring.created_at + @hiring.time.hours
+      BillingWorker.perform_async(@hiring.id, current_user.id)
```

Comment:
tại sao lại dùng perform_async mà không dùng perform_in hay perform_at

Reply:
perform_async để thực hiện jib ngay lập tức, còn in vs at là để delay ạ

Reply:
nếu gửi ngay thì cần dùng sidekiq để làm gì nhỉ?

Reply:
Vì khi ấn order thì web load sẽ khá lâu vì thực hiện nhiều request, nên em chuyển hành động send mail này vào sidekiq để web ko bị chậm ạ